_posts/2017-01-17-the-mouse-is-not-my-friend.md

```diff
@@ -0,0 +1,95 @@
+﻿---
+layout: post
+title: "The mouse is not my friend"
+date: 2017-01-17 14:54:53
+tags:
+authors: Chad Peters
+image: https://c1.staticflickr.com/2/1618/24978380044_5d4a779fcd_b.jpg
+image_url: https://www.flickr.com/photos/stavos52093/24978380044/
+image_credit: Stavos
+---
+Rather than enjoying some R&R during our recent holiday vacation, I worked on a small side project and was rewarded with an aching right arm from the extra 
+mouse time. This caused me to spend a little time thinking about how I work during the week and what I can do to minimize the pain for my future self. 
+
+There are probably half a dozen programs I use regularly. I am hoping that if I can learn keyboard shortcuts for the most common tasks I do in each, I might 
+be able to decrease significantly the amount of time I have to move my hand off of my keyboard. This will hopefully eliminate strain on my arm and it might 
+make me marginally more productive (I can use all the help I can get!)
+
+Someone else might find one or two of these useful so I will share them here. I am a Windows user and these are personal to the way I work so your mileage 
+will vary. 
+
+## Windows Shortcuts
+- `Windows Key+E`: Open File Explorer
+- `Windows Key+<anything>`: You can press the Windows Key and just start typing whatever it is you are looking for. Want to open Word? Press the Windows 
+ Key and type “Word”. Want to open the Device Manager? Press the Windows Key and type “Device”. Experiment and see what you can find. 
+- `Alt+Tab`: Cycle through all of your open Windows
+- `Windows Key+1 through 0`: Open/Switch to the respective first 10 programs on your Task Bar. 
+- `Windows Key+Arrow Up/Down/Left/Right`: This shortcut is helpful for moving your current window around. If you work with multiple monitors this will 
+ allow you to move your window to a different monitor, snap the window to different sides of your screen, maximize it or minimize it. 
+- `Shift+F10`: Right-click
+
+## Browser Shortcuts
+- `Ctrl(+Shift)+Tab`: Switch browser tabs
+- `Ctrl+1 through 9`: This shortcut will open the respective first 9 tabs. This is an especially useful shortcut if you always keep specific pages open in 
```

Comment:
In Google Chrome `Ctrl + 9` always shows the tab furthest to the right, even if 20 tabs are open.